src/legacy/status_im/communities/e2e.cljs

```diff
@@ -1,35 +1,196 @@
 (ns legacy.status-im.communities.e2e
```

Comment:
can we move this out of legacy? 
It would be nice if we can move towards right-clicking and deleting all of legacy folder :)

Reply:
UI should be using new components, for non UI we just need to move only code that is used in status 2.0